src/Common/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WordPress\AiClient\Common\Exception;
+
+use WordPress\AiClient\Exceptions\AiClientExceptionInterface;
+
+/**
+ * Exception thrown when an invalid argument is provided.
+ *
+ * This extends PHP's built-in InvalidArgumentException while implementing
+ * the AI Client exception interface for consistent catch handling.
+ *
+ * @since 0.2.0
```

Comment:
Here and throughout, always use `n.e.x.t` because we don't want to assume we know the version something will be released in.
```suggestion
 * @since n.e.x.t
```

Reply:
resolved